department-of-veterans-affairs/caseflow
# Queue | Use previous_task.assigned_at in LegacyTaskSerializer

### Description
In `LegacyTaskSerializer`, we were previously setting `previous_task.assigned_on` to the task's `assigned_on` date, rather than that of the `previous_task`.

### Acceptance Criteria 
- [ ] For `JudgeLegacyTask`s, `previous_task.assigned_on` < `assigned_on`

### Testing Plan
1. Go to `/queue` as `BVAAABSHIRE`, go to case detail
2. Start ready for dispatch flow
3. On Evaluate Decision screen, confirm "Date assigned to attorney" != "Decision submitted to judge"

Before
![screen shot 2018-08-09 at 1 45 33 pm](https://user-images.githubusercontent.com/8533004/43916064-8807b064-9bda-11e8-83f0-070b2ff0fcab.png)

After
![screen shot 2018-08-09 at 1 39 19 pm](https://user-images.githubusercontent.com/8533004/43915738-a620776c-9bd9-11e8-85b4-a1c8c2a29ebe.png)
